### ISSUE TITLE
Pythons not following host certificate chain

### ISSUE THREAD

**Issue by mcandre** [pyenv/pyenv#1687]
Pythons installed with pyenv are not using my host (macOS) trust store, breaking many things. Please make sure that all installable Pythons properly hook up to the host trust store.

**Comment by native-api** [comment]
Pyenv Pythons use Homebrew Openssl as backend.
Homebrew OpenSSL [makes a copy of the system root certificate store upon its installation](https://formulae.brew.sh/formula/openssl@1.1).
If you update the system root certificate store, you can update this copy with `brew postinstall openssl`.

**Comment by native-api** [comment]
If the above is not sufficient, there's probably a problem with a particular HTTPS server. E.g. some servers fail to provide an intermediate certificate in the certificate chain as required by the TLS standard.

**Comment by native-api** [comment]
TL;DR not Pyenv's issue AFAICS.
